cvat/apps/dataset_manager/project.py

```diff
@@ -158,7 +158,7 @@ def import_dataset(self, dataset_file, importer, **options):
         with TemporaryDirectory(dir=temp_dir_base) as temp_dir:
             try:
                 importer(dataset_file, temp_dir, project_data, self.load_dataset_data, **options)
-            except DatasetNotFoundError as not_found:
+            except (DatasetNotFoundError, CvatDatasetNotFoundError) as not_found:
```

Comment:
```suggestion
            except DatasetNotFoundError as not_found:
```

As the class is inherited, no extra handling is needed here.